sdk/src/test/java/io/dapr/client/DaprHttpTest.java

```diff
@@ -91,35 +111,79 @@ public void invokeMethodWithHeaders() throws IOException {
         headers.put("header1","value1");
 
         mockInterceptor.addRule()
-                .get("http://localhost:3500/v1.0/get?header1=value1&header=value")
```

Comment:
Do we have any other test for params?

Reply:
I added another one for parameters. When i fix the styling issue, i will push that test as well.